File: slides/omp/code/hello.cc
#include <omp.h>
#include <cstdio>
using std::printf;
int main()
{
#pragma omp parallel
{
 int nt = omp_get_num_threads();
 int id = omp_get_thread_num();
 printf("hello world from %i "
        "of %i\n", id, nt);
}
}

File: slides/omp/code/hello.f90
program hello_world
use omp_lib
implicit none
integer :: nt, id
!$omp parallel
nt = omp_get_num_threads()
id = omp_get_thread_num()
print *, "hello world from ", &
         id, " of ", nt
!$omp end parallel
end program hello_world
